domains/wapiy.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Portfolio website",
+    "repo": "https://github.com/Wapiy-y/abdullahwafiy",
+    "owner": {
+        "username": "Wapiy",
+        "email": "wafiyabdullah39@gmail.com"
+    },
+    "record": {
+        "CNAME": "abdullahwafiy.vercel.app/"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "abdullahwafiy.vercel.app"
```
also your website looks like a template and is incomplete, please add more content